app/javascript/layouts/AssistanceLayout.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import Layout from "./Layout"
+import { PageHeader, SidebarBlock, t } from "@bloom-housing/ui-components"
+import { renderInlineMarkup } from "../util/languageUtil"
+import { ConfigContext } from "../lib/ConfigContext"
+
+import "./AssistanceLayout.scss"
+
+export interface Props {
+  children: React.ReactNode
+  title: string
+  subtitle: string
+}
+
+const AssistanceLayout = ({ children, title, subtitle }: Props) => {
+  const { getAssetPath } = React.useContext(ConfigContext)
+
+  return (
+    <Layout title={title}>
+      <PageHeader
+        title={title}
+        subtitle={subtitle}
+        inverse
+        backgroundImage={getAssetPath("bg@1200.jpg")}
+      />
+      <article className="flex flex-wrap flex-col md:flex-row relative max-w-5xl m-auto w-full">
+        <div className="w-full md:w-2/3" data-test-id="assistance-main-content">
+          {children}
+        </div>
+        <SidebarBlock
+          className="w-full md:w-1/3 text-gray-950 md:border-l border-t md:border-t-0 border-gray-450 p-6 mx-0"
+          title={t("footer.contact")}
+        >
+          <span className="text-gray-950">
+            <p className="mb-4">
+              <span className="font-semibold">
+                {t("assistance.contact.questionsAboutListings.title1")}
+              </span>
+              <br /> {renderInlineMarkup(t("assistance.contact.questionsAboutListings.subtitle"))}
+            </p>
+            <p className="mb-4">
+              <span className="font-semibold">{t("assistance.contact.helpLine.title1")}</span>
+              <br /> {renderInlineMarkup(t("assistance.contact.helpLine.subtitle"))}
+            </p>
+            <div>
+              <span className="font-semibold">
+                {t("assistance.contact.questionsAboutPrograms.title1")}
+              </span>
+              <br />
+              <span>
+                {renderInlineMarkup(t("assistance.contact.questionsAboutPrograms.contactWebsite"))}
+              </span>
+              <span className="mt-1">
+                {renderInlineMarkup(t("assistance.contact.questionsAboutPrograms.contactEmail"))}
```

Comment:
looks like we're missing some paragraphs?
<img width="983" alt="image" src="https://user-images.githubusercontent.com/98352600/222861462-90b2e64c-5178-4ff9-bf11-473dd9e436d2.png">


also this one is a nit, but i do like the whitespacing in the mock between lines, particularly the bold first lines and the rest of its section.

Reply:
Thank you for catching this!